### ISSUE TITLE
[A11y] Opening information popup does not announce information, popup can not be reached through narrator

### ISSUE THREAD

**Issue by chingucoding** [jenius-apps/ambie#53]
When opening the information popup, the information is not announced nor can it be reached through narrator.

**Comment by dpaulino** [comment]
How does one make the information in the pop reachable by narrator?

**Comment by dpaulino** [comment]
Is it AutomationProperties.Name?

**Comment by chingucoding** [comment]
The issue I saw is that you cannot navigate to the flyout itsself. Navigating with the arrow keys doesn't allow you to get to the text of the flyout.

**Comment by dpaulino** [comment]
I see. I think you can't do that because the textblocks are not focusable. Is there a UIA way of making them focusing only when screen reader is turned on? Or do I have to add a tabstop for these textblocks? Any chance you know?

**Comment by chingucoding** [comment]
I would raise a WindowOpened event which should let narrator announce the content of the popup. TeachingTip does the following when light dismiss is enabled:

```c++
    // We only report as a window when light dismiss is enabled.
    if (winrt::get_self<TeachingTip>(GetTeachingTip())->IsLightDismissEnabled() &&
        winrt::AutomationPeer::ListenerExists(winrt::AutomationEvents::WindowOpened))
    {
        RaiseAutomationEvent(winrt::AutomationEvents::WindowOpened);
    }
```

**Comment by chingucoding** [comment]
Any updates on this and #54 @dpaulino?

**Comment by dpaulino** [comment]
This is now obsolete. The about pop up no longer exists
